src/Editor.css

```diff
@@ -139,117 +156,122 @@
   min-height: 100%;
   left: 0;
   top: 0;
-  background-color: rgba(0, 0, 0, 0.5);
+  background-color: rgba(0, 0, 0, 10%);
+  backdrop-filter: blur(3px);
+
+  animation: fade-in var(--easing-medium) forwards;
+}
+
+@keyframes pop-in {
+  from {
+    transform: translate(-50%, -50%) scale(0.9);
+    opacity: 0;
+  }
+
+  to {
+    transform: translate(-50%, -50%) scale(1);
+    opacity: 1;
+  }
 }
 
 ._legoEditor_modal_root {
-  opacity: 1;
-  background-color: var(--color-white);
-  width: 60%;
-  top: 10%;
-  left: 20%;
-  margin: 0 30px;
-  padding: 15px;
-  border-radius: 5px;
   display: flex;
   flex-direction: column;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 475px;
+  max-width: 100%;
+  max-height: calc(100vh - 100px);
+  padding: 35px;
+  background: var(--color-white);
+  outline: none;
+  overflow-y: auto;
+  box-shadow: var(--shadow-sm);
+  border-radius: var(--border-radius-lg);
+  animation: pop-in var(--easing-medium) forwards;
```

Comment:
Haven't tested this locally myself, but I assume that this has been tested with devtools for different screen sizes?

Reply:
Yes, it was tested. It's the same styling as in lego